tests/unit/sagemaker/cli/compatibility/v2/modifiers/test_framework_version.py

```diff
@@ -21,135 +21,215 @@
+        if versions:
+            keywords["framework_version"] = self.framework_version
+            if self.py_version_for_model:
+                keywords["py_version"] = self.py_version
+        return _format_templates(keywords, self.model_templates)
+
+
+def _format_templates(keywords, templates):
+    args = ", ".join(
+        "{key}='{value}'".format(key=key, value=value) for key, value in keywords.items()
+    )
+    return [template.format(args) for template in templates]
+
+
+TEMPLATES = [
+    Template(
+        framework="TensorFlow",
+        framework_version="1.11.0",
+        py_version="py2",
+        framework_templates=[
+            "TensorFlow({})",
+            "sagemaker.tensorflow.TensorFlow({})",
+            "sagemaker.tensorflow.estimator.TensorFlow({})",
+        ],
+        model_templates=[
+            "TensorFlowModel({})",
+            "sagemaker.tensorflow.TensorFlowModel({})",
+            "sagemaker.tensorflow.model.TensorFlowModel({})",
+        ],
+        py_version_for_model=False,
+    ),
+    Template(
+        framework="MXNet",
+        framework_version="1.2.0",
+        py_version="py2",
+        framework_templates=[
+            "MXNet({})",
+            "sagemaker.mxnet.MXNet({})",
+            "sagemaker.mxnet.estimator.MXNet({})",
+        ],
+        model_templates=[
+            "MXNetModel({})",
+            "sagemaker.mxnet.MXNetModel({})",
+            "sagemaker.mxnet.model.MXNetModel({})",
+        ],
+    ),
+    Template(
+        framework="Chainer",
+        framework_version="4.1.0",
+        py_version="py3",
+        framework_templates=[
+            "Chainer({})",
+            "sagemaker.chainer.Chainer({})",
+            "sagemaker.chainer.estimator.Chainer({})",
+        ],
+        model_templates=[
+            "ChainerModel({})",
+            "sagemaker.chainer.ChainerModel({})",
+            "sagemaker.chainer.model.ChainerModel({})",
+        ],
+    ),
+    Template(
+        framework="PyTorch",
+        framework_version="0.4.0",
+        py_version="py3",
+        framework_templates=[
+            "PyTorch({})",
+            "sagemaker.pytorch.PyTorch({})",
+            "sagemaker.pytorch.estimator.PyTorch({})",
+        ],
+        model_templates=[
+            "PyTorchModel({})",
+            "sagemaker.pytorch.PyTorchModel({})",
+            "sagemaker.pytorch.model.PyTorchModel({})",
+        ],
+    ),
+    Template(
+        framework="SKLearn",
+        framework_version="0.20.0",
+        py_version="py3",
+        framework_templates=[
+            "SKLearn({})",
+            "sagemaker.sklearn.SKLearn({})",
+            "sagemaker.sklearn.estimator.SKLearn({})",
+        ],
+        model_templates=[
+            "SKLearnModel({})",
+            "sagemaker.sklearn.SKLearnModel({})",
+            "sagemaker.sklearn.model.SKLearnModel({})",
+        ],
+        py_version_for_model=False,
+    ),
+]
+
+
+def get_constructors(versions=False, image=False):
```

Comment:
I believe it's less Pythonic for a function to start with "get" - it's considered redundant?

Reply:
removing